### PR TITLE
Add help text and game restart warning

### LINKED ISSUE #925: /wa help
Buds mentioned this in passing, and I don’t want this to be forgotten.

Basically, we should add a help option to the WeakAuras slash command, which prints out all the subcommands which are available to them.

e.g.
```
> /wa help
Usage:
/wa help - show this message
/wa pstart - begin a fresh profile
/wa pstop - finish profiling
/wa pprint - bring up a window with the results from the most recent profile
```

### PR DESCRIPTION
**Pull request by Stanzilla**
Decisions I made:
- lower the slash command message in case people want to do /wa hElP etc 
- Display Discord URL but not GitHub because Discord can pre-filter most problems
- Display the version mismatch warning 1s after load so all the other addons don't spam it away, maybe display on options open as well?

Closes #925

### CODE REVIEW

**Comment by mrbuds** [comment]
#925 for reference

File: WeakAuras/Init.lua [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
 
 WeakAuras.prettyPrint = function(msg)
   print(WeakAuras.printPrefix .. msg)
 end
 
+WeakAuras.versionMismatchPrint = function()
+  WeakAuras.prettyPrint("You need to restart your game client to complete the WeakAuras update!")
+end
+
+if versionString ~= versionStringFromToc and versionString ~= "Dev" then
```
Comment (emptyrivers) [inline WeakAuras/Init.lua]: This will cause a lot of false positives whenever there is an update which doesn’t add any new files.

Comment (Stanzilla) [reply WeakAuras/Init.lua]: Yeah well, doesn't hurt either?

Comment (emptyrivers) [reply WeakAuras/Init.lua]: Well, it’s probably fine. People have awful hygiene when it comes to updating addons anyways.

**Pull request merged**